cardano-node/src/Cardano/CLI/Tx/Generation.hs

```diff
@@ -49,6 +51,11 @@ import           Data.Text (Text)
 import qualified Data.Text as T
 import           Data.Time.Clock (DiffTime, picosecondsToDiffTime)
 import           Data.Word (Word8, Word32, Word64)
+import           Network.HTTP.Client (Request (..), RequestBody (..),
```

Comment:
Please follow the existing import indentation style of this file.

Reply:
Sorry, didn't understand what I should fix..

```
import           Network.HTTP.Client (Request (..), RequestBody (..),
                                      defaultManagerSettings, httpLbs,
                                      newManager, parseRequest, responseBody,
                                      responseStatus)
```

Should I replace it by:

```
import           Network.HTTP.Client (Request (..), RequestBody (..),
                     defaultManagerSettings, httpLbs,
                     newManager, parseRequest, responseBody,
                     responseStatus)
```

?

Reply:
Yes, that is closer to the existing import style. It can probably be done it 3 lines rather than 4.